test/dane-cross.in

```diff
@@ -0,0 +1,113 @@
+# Copyright 2016-2016 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
2016 ? twice ?

Reply:
Copied from the original danetest.in, but I'll change it to 2016-2021, both in this PR #16473.

Reply:
That's right: [danetest.in](https://github.com/openssl/openssl/blob/master/test/danetest.in)